dcgtc/dgrants
# Custom Top LVL Grant Filter

closes #253 

to test:

- open app/src/CustomGrantIndexList.json and add custom index list I.E. [0, 2, 3, 5]
- check the app to see that these are now the only grants shown (and can be sorted)
- when list is empty, all grants are shown

Review:

- which folder to have CustomGrantIndexList.json in (current location: app/src/)
- consider whether this is fine or shall we store in IPFS or ENV variable
